### PR TITLE
[2.5.7] Introducing aks-operator Helm chart

### PR DESCRIPTION
**Pull request by mjura**

### CODE REVIEW

File: charts/rancher-aks-operator/1.0.100/questions.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.5.0-alpha1
```
Comment (rmweir) [inline charts/rancher-aks-operator/1.0.100/questions.yaml]: I believe this should be 2.5.7-alpha1.

File: charts/rancher-aks-operator/1.0.100/templates/clusterrole.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: aks-operator
+  namespace: cattle-system
+rules:
+  - apiGroups: ['']
+    resources: ['secrets']
+    verbs: ['get', 'list', 'create', 'watch']
```
Comment (rmweir) [inline charts/rancher-aks-operator/1.0.100/templates/clusterrole.yaml:9]: Pending changes to aks-operator we may be able to remove some or all of these secrets permissions in the future.

**Review by rmweir — APPROVED** [review]
LGTM. Keep track of permissions used in aks-operator so we can remove anything not needed. Left a comment regarding this. Will need to replace image tag once there is actually an aks-operator tag available. Can probably wait to merge until that is done.

**Review by cbron — COMMENTED** [review]
lgtm, waiting for 2.5.6 release

**Review by cbron — APPROVED** [review]

**Pull request merged**